tcf_website/views/review.py

```diff
@@ -123,3 +86,23 @@ def get_object(self):  # pylint: disable=arguments-differ
             raise PermissionDenied(
                 "You are not allowed to delete this review!")
         return obj
+
+
+@login_required
+def edit_review(request, review_id):
+    """Review modification view."""
+    review = get_object_or_404(Review, pk=review_id)
+    if review.user != request.user:
+        raise PermissionDenied('You are not allowed to edit this review!')
```

Comment:
Attempting to edit a review you don't own by manually changing the URL results in this error screen:

![image](https://user-images.githubusercontent.com/55100084/104984650-22b71380-59dd-11eb-89fd-327e41e289e4.png)

This isn't terribly important because the key thing is that the permission checking works and it's unlikely anyone would try, but is there a easy way to more elegantly show this error?

Reply:
This should be done as part of #274:
> Create 404/403/500 pages that looks better and provides more information to the user.

Typical end users won't try that, so I believe this is outside the scope of this PR.